runtime.go

```diff
@@ -34,3 +35,26 @@ func (rt *Runtime)GetManager() service.Manager{
 	return rt.svc
 }
 
+func EnvironmentSpecificConfig(v *viper.Viper, env string) *viper.Viper {
```

Comment:
where are you using that??
it shouldn't be part of this pull request, 
revert back.